site/src/sphinx/server-access-log.rst

```diff
@@ -306,3 +306,50 @@ You can specify your own log writer which implements a ``Consumer`` of :api:`Req
         // Write your access log with the given RequestLog instance.
         ....
     });
+
+
+Customizing an access logger
+----------------------------
+
+Armeria use an access logger depending on the reversed domain name of an each VirtualHost by default.
+e.g) `com.linecorp.armeria.logging.access.com.example` for `*.example.com`
+     `com.linecorp.armeria.logging.access.com.linecorp` for `*.linecorp.com`
+
+However, You can specify your own policy or your own logger for a VirtualHost.
+
+.. code-block:: java
+
+    ServerBuilder sb = new ServerBuilder();
+
+    // Using the specific logger name.
+    sb.accessLogger("Write your access logger name you want.");
+
+    // Using your own logger.
+    Logger logger = ...;
+    sb.accessLogger(Logger);
+
+    // Using the mapper which sets an access logger with the given VirtualHost instance.
+    sb.accessLogger(virtualHost -> {
+        // Write your access logger with the given VirtualHost instance.
+        ....
+    });
+
+    // You can use VirtualHostBuilder.accessLogger to specify your own logger for the VirtualHost.
+    sb.withVirtualHost("*.example.com")
+    .accessLogger("Write your access logger name you want.")
+    .and()
+    ....
+
+    // Using your own logger.
+    Logger logger = ...;
+    sb.withVirtualHost("*.example2.com")
+    .accessLogger(Logger)
+    .and()
+    ....
+
+    // Using the mapper which sets an access logger with the given VirtualHost instance.
+    sb.withVirtualHost("*.example3.com")
+    .accessLogger(virtualHost -> {
+        // Write your acess logger with the given VirtualHost instance.
+        ....
+    }).and()
```

Comment:
Ditto + indentation

- The last `and()` is probably redundant and we could add `...`